### DISCUSSION TITLE
Provisional rating indicator

### DISCUSSION THREAD

**Discussion by m1ntleaf** [osu-tournament-rating/otr-web#137] [Ideas]
A while ago, Cytusine (https://github.com/cytusine0) made a discussion about generating initial ratings. Perhaps it would be useful to add a tag on players to indicate lack of data (e.g. less than 5 tournaments played). I do not currently have access to the state of the website, but it could be useful to consider.

- MintLeaf

**Comment by hburn7** [comment]
We actually do have this as a feature currently, provisional status. I think the wiki does not have this documented at this time.

**Comment by m1ntleaf** [comment]
Okay. :) Is there a way for me to contribute to the wiki, or does it have some level of protection?

**Reply by myssto** [reply]
You can contribute to the wiki by creating a fork of the [wiki](https://github.com/osu-tournament-rating/otr-wiki), making your changes, and then creating a pull request. Be sure to read our [contribution guidelines](https://github.com/osu-tournament-rating/otr-wiki/tree/master/contribution). Those rules apply to wiki as well!

**Reply by m1ntleaf** [reply]
Oh, thanks!!

**Reply by hburn7** [reply]
Before adding to the wiki, definitely let us know what you would like to work on so that we are aware.
